apps/aecore/src/aec_headers.erl

```diff
@@ -66,6 +68,7 @@
           root_hash    = <<0:?STATE_HASH_BYTES/unit:8>>        :: state_hash(),
           signature    = <<0:?BLOCK_SIGNATURE_BYTES/unit:8>>   :: block_signature(),
           txs_hash     = <<0:?TXS_HASH_BYTES/unit:8>>          :: txs_hash(),
+          pof          = no_fraud                              :: aec_pof:pof(),
```

Comment:
`aec_pof:pof()` should be a map, and here the type should be `no_fraud | aec_pof:pof()`